From f8955b286554446679e75d283b96fe3ead00e64b Mon Sep 17 00:00:00 2001
From: Vlad Yasevich <vladislav.yasevich@hp.com>
Date: Fri, 23 Oct 2009 16:18:34 -0400
Subject: [PATCH 04/53] apps: Fix compiler warnings in sctp_darn

Code was calling fprintf() without a format, but with an argument to
evaluate.  That's really pointless as it doesn't produce anything
and generates a compiler warning.

Signed-off-by: Vlad Yasevich <vladislav.yasevich@hp.com>
---
 src/apps/sctp_darn.c | 12 ++++--------
 1 file changed, 4 insertions(+), 8 deletions(-)

diff --git a/src/apps/sctp_darn.c b/src/apps/sctp_darn.c
index 61673a2..3d76730 100644
--- a/src/apps/sctp_darn.c
+++ b/src/apps/sctp_darn.c
@@ -363,8 +363,7 @@ parse_arguments(int argc, char *argv[]) {
 				 * from trying to allocate more.
 				 */
 				fprintf(stderr, "No memory to add ");
-				fprintf(stderr, optarg);
-				fprintf(stderr, "\n");
+				fprintf(stderr, "%s\n", optarg);
 				exit(2);
 			}
 			bindx_add_addrs = tmp_addrs;
@@ -381,8 +380,7 @@ parse_arguments(int argc, char *argv[]) {
 				 * from trying to allocate more.
 				 */
 				fprintf(stderr, "No memory to add ");
-				fprintf(stderr, optarg);
-				fprintf(stderr, "\n");
+				fprintf(stderr, "%s\n", optarg);
 				exit(2);
 			}
 			bindx_rem_addrs = tmp_addrs;
@@ -397,8 +395,7 @@ parse_arguments(int argc, char *argv[]) {
 				 * from trying to allocate more.
 				 */
 				fprintf(stderr, "No memory to add ");
-				fprintf(stderr, optarg);
-				fprintf(stderr, "\n");
+				fprintf(stderr, "%s\n", optarg);
 				exit(2);
 			}
 			connectx_addrs = tmp_addrs;
@@ -1755,8 +1752,7 @@ get_bindx_addr(char *in, int *count)
 		 * from trying to allocate more.
 		 */
 		fprintf(stderr, "No memory to add ");
-		fprintf(stderr, in);
-		fprintf(stderr, "\n");
+		fprintf(stderr, "%s\n", in);
 		exit(2);
 	}
 	return tmp_addrs;
-- 
1.7.11.7

